schemas/context/profile-privacy.schema.json

```diff
@@ -0,0 +1,85 @@
+{
+  "meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-privacy",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Opt In/Out Signals",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "title": "Profile Opt Out",
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "Profile Optout Mixin",
+  "definitions": {
+	"profile-privacy": {
+		"xdm:optOutConsent": {
+			"type": "object",
+			"title": "consent for optout",
+			"description": "Level for extensibility of consents for different opt out types",
+			"properties": {
+				"xdm:privacyOptouts": {
```

Comment:
The casing of `optOut` is inconsistent throughout the schema. We should align on `optOut`.